lib/reporting/identity_verification_report.rb

```diff
@@ -300,7 +300,7 @@ def query
           , coalesce(properties.event_properties.success, 0) AS success
         #{issuers.present? ? '| filter properties.service_provider IN %{issuers}' : ''}
         | filter name in %{event_names}
-        | filter (name = %{usps_enrollment_status_updated} and properties.event_properties.passed = 1)
+        | filter (name = %{usps_enrollment_status_updated} and properties.event_properties.passed = 1 and properties.event_properties.tmx_status = 'threatmetrix_pass')
```

Comment:
is `tmx_status` always on that event? Like do we need to add an `ispresent()` guard on this so that old events before this applied will still be filtered correctly?

Reply:
I went for something different but equivalent in this commit: https://github.com/18F/identity-idp/pull/10798/commits/62556c7ef4b1c96b9f8c1d275f5a9bff2c3dd92d